### ISSUE TITLE
Website `opensourcerover.jpl.nasa.gov` is down

### ISSUE THREAD

**Issue by maheshrijal** [nasa-jpl/open-source-rover#413]
Based on the error, perhaps it is to do with the AWS account permissions. 

![image](https://github.com/nasa-jpl/open-source-rover/assets/62394512/3e30196e-aa60-48b7-ae49-d15cc6877d80)

**Comment by eddmakesio** [comment]
Yep, similar issue on Safari

```
KMS.UnrecognizedClientExceptionNo account found for the given parametersC87WCZMAYJZ7BX1PKSzn83Z7gWzFJ46RcolBvQSYR97K/w5LGV1rtYSWwNz3P1CkIMKnPsPx8CiQvIXaZ0Uq6zUuJ9A=
```

**Comment by Achllle** [comment]
Unfortunately due to spending cuts at JPL, the website is no longer paid for for the time being. I'm hoping it comes back but until then the links will have to be removed. If anyone would like to tackle that, please create a PR

**Comment by maheshrjl** [comment]
@Achllle is hosting the page on GitHub pages feasible?

**Comment by Achllle** [comment]
I don't know. I'd likely have to get access to the website's source code and I'm not sure if that's worth the effort given that success isn't guaranteed.

**Comment by maheshrjl** [comment]
I think we already have everything required for deployment in the repository. I was able to spin up the website using mkdocs on my local.

![image](https://github.com/nasa-jpl/open-source-rover/assets/62394512/343e5834-91ac-42dd-a27f-beca84deaed7)

![image](https://github.com/nasa-jpl/open-source-rover/assets/62394512/4df7293e-9f94-4812-a2bb-09f0d9fd2e70)

But, deploying to pages would also require modifying the DNS records for `opensourcerover.jpl.nasa.gov`. I see it is currently pointed to a CloudFront distribution.

I can submit a PR for deployment to pages if required. Thanks :)

**Comment by ericjunkins** [comment]
The information that is on the github repo and what you have displayed in that mkdocks is not the same as the website that was previously hosted at opensourcerover.jpl.nasa.gov, that was like an educational site about getting to see an animation that was made about the rover, as well as highlighting some of the specifics of how it works.

**Comment by eddmakesio** [comment]
Out of interest @ericjunkins / @Achllle is this something that could be crowdfunded/sponsored? Sad to see educational content go offline due to short term funding. Submitted a PR for the link anyway. 

https://github.com/nasa-jpl/open-source-rover/pull/414

**Comment by Achllle** [comment]
@mikcox Since @maheshrjl seemingly was confusing the JPL website with our `mkdocs` documentation, I doubt that we can use the source code to host the website ourselves. I think the only way forward is to have JPL reinstate the domain - probably at little or no cost to them.

**Comment by Achllle** [comment]
closed by #414, conversation stale
